dpytools/http/upload/utils.py

```diff
@@ -69,6 +69,9 @@ def _generate_upload_new_params(
     if title is None:
         title = file_path.stem
 
+    if is_publishable is None:
+        is_publishable = False
```

Comment:
pop these as defaults in the function rather than these statements

(or 
title = "sadfsadf" if title is None)